articles/azure-stack/azure-stack-sql-resource-provider-deploy.md

```diff
@@ -101,11 +101,22 @@ Change the default account information and passwords as needed for your Azure St
 
 
 ```powershell
-# Install the AzureRM.Bootstrapper module, set the profile and install the AzureStack module
+# Azure Stack version 1901 or later, only install the two modules below:
+# Install and import the API Version Modules required by Azure Stack into the current PowerShell session.
+
+Install-Module AzureRM -RequiredVersion 2.4.0
+Install-Module -Name AzureStack -RequiredVersion 1.7.1
+```
+
+```powershell   
+# Install the AzureRM.Bootstrapper module, set the profile and install the AzureStack module for Azure Stack version 1811 or earlier.
 # Note that this might not be the most currently available version of Azure Stack PowerShell
 Install-Module -Name AzureRm.BootStrapper -Force
 Use-AzureRmProfile -Profile 2018-03-01-hybrid -Force
 Install-Module -Name AzureStack -RequiredVersion 1.6.0
+```
+> [!Note]  

```

Comment:
@bganapa Did you mean to place this note here?

Reply:
SQL RP still needs the old version for 1901+ bits as well, so their instructions are already correct.. we should not change this file